client/js/restaurants_api.js

```diff
@@ -1169,6 +1349,57 @@ function create_reviews_table(reviewsResponse)
 	
 }
 
+function create_organizations_reviews_table(reviewsResponse)
+{
+	reviewsResponse= JSON.parse(reviewsResponse);
+
+	if (reviewsResponse.length < 1)
+	{
+		document.getElementById("reviews_table_body").innerHTML = "<tr>No reviews are available.</tr>";
+	    return;
+
+	}
+
+	//clean previous table content
+	var myNode = document.getElementById("reviews_table_body");
+		while (myNode.firstChild) {
+		    myNode.removeChild(myNode.lastChild);
+		}
+	cancel_reservation_url_base = "";
+	for (var j=0, lim=reviewsResponse.length; j < lim; j++)
+	{
+
+		var row = document.createElement("TR");
+
+		var name = document.createElement("TD");
+		name.innerHTML = reviewsResponse[j]["itemReviewed"]["name"];
+		name.setAttribute("class","col-xs-4");
+		row.appendChild(name);
+
+		var rating = document.createElement("TD");
+		rating.innerHTML =  reviewsResponse[j]["reviewRating"]["ratingValue"];
+		rating.setAttribute("class","col-xs-2");
+		row.appendChild(rating);
+
+
+		var view = document.createElement("TD");
+		view.innerHTML = '<a href="javascript:view_review('+
```

Comment:
upssss please do not use javascript URLs that is old, very old style add proper event handlers